ui/nodeman-ui-config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "events": {
```

Comment:
whats the reason for this?

Reply:
@labibc nodemon is the thing that watches our files for changes. This file is to override the configuration for nodemon. We're saying "when you restart/crash, instead of doing nothing, run the ui-build.sh script". That script then creates a build.